api/v1alpha1/toolchainconfig_types.go

```diff
@@ -231,20 +239,17 @@ type RegistrationServiceConfig struct {
 	LogLevel *string `json:"logLevel,omitempty"`
 
 	// Namespace specifies the namespace in which the registration service and host operator is running
+	// Consumed by host operator and set as env var on registration-service deployment
 	// +optional
 	Namespace *string `json:"namespace,omitempty"`
 
 	// RegistrationServiceURL is the URL used to a ccess the registration service
 	// +optional
 	RegistrationServiceURL *string `json:"registrationServiceURL,omitempty"`
 
-	// Keeps parameters necessary for the registration service analytics config
+	// Replicas specifies the number of replicas to use for the registration service deployment
 	// +optional
-	Analytics RegistrationServiceAnalyticsConfig `json:"analytics,omitempty"`
-
-	// Keeps parameters necessary for the registration service authentication config
-	// +optional
-	Auth RegistrationServiceAuthConfig `json:"auth,omitempty"`
+	Replicas *string `json:"replicas,omitempty"`
```

Comment:
Shouldn't it be an int32 with `1` as the default value?
```suggestion
	Replicas *int32 `json:"replicas,omitempty"`
```

Reply:
yes, for some reason I thought we used string but int32 makes more sense

Reply:
Regenerated the files and updated https://github.com/codeready-toolchain/host-operator/pull/495 as well